contracts/swap/swap.go

```diff
@@ -164,16 +174,39 @@ func (s simpleContract) CashChequeBeneficiary(opts *bind.TransactOpts, beneficia
 	return result, receipt, nil
 }
 
-// LiquidBalance returns the LiquidBalance (total balance in Wei - total hard deposits in Wei) of the chequebook
+// LiquidBalance returns the LiquidBalance (total balance in ERC20-token - total hard deposits in ERC20-token) of the chequebook
 func (s simpleContract) LiquidBalance(opts *bind.CallOpts) (*big.Int, error) {
 	return s.instance.LiquidBalance(opts)
 }
 
+//Token returns the address of the ERC20 contract, used by the chequebook
+func (s simpleContract) Token(opts *bind.CallOpts) (common.Address, error) {
```

Comment:
Up until now, it has been a convention to use this parameter, also for calls to the blockchain (see LiquidBalance).

We don't use these parameters now, but in the future they might come in handy.

Options are:
```golang
	Pending     bool            // Whether to operate on the pending state or the last known one
	From        common.Address  // Optional the sender address, otherwise the first account is used
	BlockNumber *big.Int        // Optional the block number on which the call should be performed
	Context     context.Context // Network context to support cancellation and timeouts (nil = no timeout)
```

Reply:
Please **always** keep opts as input when doing any blockchain interaction as it contains the context which we will have to set to some proper value at some point.

Reply:
thanks for the clarification guys